src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="img__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover">Laptops & Computers
```

Comment:
not so good as it can be)
please check code style for html in your [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) (13 point)
no closing arrow on the line with attribute

```
<tag
  attr1
  ...
>
 content
</tag>
```